packages/components/addon/components/hds/app-footer/copyright.hbs

```diff
@@ -0,0 +1,10 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{! TODO: use new Text component? }}
```

Comment:
mmm... yes? 😀

Reply:
Thanks, I added that note before it had been merged in. Just refactored.